Toolchain/Patches/gcc/ReadMe.md

```diff
@@ -64,72 +64,3 @@ it statically into LibC/our shared objects. However, the build system
 forces no-pic/pie instead.
 
 This hack is from https://gcc.gnu.org/bugzilla/show_bug.cgi?id=58638
-
-## `0008-Include-safe-ctype.h-after-C-standard-headers-to-avo.patch`
```

Comment:
Hmm, is it not a problem anymore? Or did you just take Daniel's patches and he silently dropped this?

Reply:
No this one was a cherry-picked upstream commit, so with the new version its already included